src/ngraph/frontend/onnx_import/op/supported_ops.md

```diff
@@ -0,0 +1,137 @@
+## Supported Ops:
+
+Generally nGraph doesn't support tensors of types:
+
+* `string`,
+* `float16`,
+* `complex64`,
+* `complex128`.
+
+Value in `()` _brackets_ means that this op is available since specific opset version.
```

Comment:
Value in `()` _brackets_ indicates that this op is available and supported as of the specific opset version.

Reply:
I used `()` just to notice what operators have changed in ninth ONNX Standard opset version. These operators usually are not supported yet.